vtysh/vtysh.c

```diff
@@ -4721,6 +4721,22 @@ static int vtysh_connect(struct vtysh_client *vclient)
 		close(sock);
 		return -1;
 	}
+	/* Setting the RCV buffer size to 16MB to increase the buffer size
+	 * the socket can hold after receving from other process
+	 */
+	uint32_t rcvbufsize = RCV_BUF_MAX;
```

Comment:
please declare the locals at the top, follow the existing style.

Reply:
Will do